src/components/create_cluster/index.js

```diff
@@ -380,8 +363,8 @@ class CreateCluster extends React.Component {
 
             <div>
               <div className='row section'>
-                <div className='col-12'>
-                  <h3 className='table-label'>Worker Node Configuration</h3>
+                <div className='col-3'>
+                  <h3 className='table-label'>Instance Type</h3>
```

Comment:
"Instance Type" is AWS-specific terminology. Fo Azure it would be "VM Size". For KVM we could leave it at "Worker Node Configuration".